### PR TITLE
Adding redux and localStorage support - resolved #8

### LINKED ISSUE #8: Add Redux to project

### PR DESCRIPTION
**Pull request by jhonandresmedina**

### CODE REVIEW

File: src/actions/app.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import actionTypes from '../constants/actionTypes';
```
Comment (filgaia) [inline src/actions/app.js]: Maybe name it as `participants.js` so it's not named `app.js`?

File: src/constants/index.js [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-    { index: 7, name: 'Josimar', description: "Las burras" },
-    { index: 8, name: 'Ximena', description: "La crocssss" },
-    { index: 9, name: 'Jose', description: "El chico futbolista" },
-    { index: 10, name: 'Astrid', description: "La nueva BA, no no no no, bueno si" },
-    { index: 11, name: 'Cristhian', description: "El super crack" },
-    { index: 12, name: 'Fabricio', description: "El piñin" },
-    { index: 13, name: 'Leo', description: "El chico automation" },
-    { index: 14, name: 'Nestor', description: "El mejor UX" },
-    { index: 15, name: 'Nacho', description: "El mejor VD" }
+    { index: 1, name: 'Alejandro', description: "Web UI Developer" },
```
Comment (filgaia) [inline src/constants/index.js]: Formalizing stuff? 
Still, we can think in not uploading this in the future as sensitive information and let the upload handle it

File: src/reducers/app.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import actionTypes from '../constants/actionTypes';
```
Comment (filgaia) [inline src/reducers/app.js]: Same here, maybe name it `participants`?

File: src/actions/app.js (hunk 1 above)
Comment (jhonandresmedina) [reply src/actions/app.js]: got it!

File: src/reducers/app.js (hunk 3 above)
Comment (jhonandresmedina) [reply src/reducers/app.js]: got it

File: src/constants/index.js (hunk 2 above)
Comment (jhonandresmedina) [reply src/constants/index.js]: got it

**Pull request merged**